modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc
+
+[id="odc-using-the-developer-catalog-to-add-services-or-components_{context}"]
+= Using the Developer Catalog to add services or components to your application
+
+Developer Catalog enables you to deploy applications and services based on Operator backed services such as Databases, Builder Images, and Helm Charts. The Developer Catalog contains a collection of application components, services, event sources, or source-to-image builders that you can add to your project. Cluster administrators can customize the content made available in the catalog.
+
+The following procedure shows you an example of how to add a *MariaDB* database service from the Developer Catalog.
+
+. In the *Developer* perspective, navigate to the *+Add -> From Catalog* to view all the available services in the *Developer Catalog*.
+. Under *All Items* select the kind of service or the component you need to add to your project. For this example, select *Databases* to list all the database services and then click on *MariaDB* to see the details for the service.
++
+.Developer Catalog
```

Comment:
I don't think you need either of these screenshots. They don't add significant value and require maintenance.